Pipfile

```diff
@@ -4,6 +4,8 @@ verify_ssl = true
 name = "grafana_api"
 
 [packages]
+requests = "*"
+mock = "*"
```

Comment:
You don't need to specify the `mock`, it's not a package requirements, but a test requirements. Also, it's Python 2 only. Check the `[dev-packages]` section.